components/typography/stories/template.js

```diff
@@ -106,7 +112,413 @@ export const Template = (args = {}, context = {}) => {
+			content: InternationalizedRegularBoldItalicGroup({
+				...args,
+				semantics: "heading",
+				lang: "he",
+				internationalizedContent: HebrewInternationalizedContent,
+			}, context)
+		})}
+		${Container({
+			direction: "column",
+			heading: "Hebrew (Heavy)",
+			content: InternationalizedRegularBoldItalicGroup({
+				...args,
+				semantics: "heading",
+				lang: "he",
+				internationalizedContent: HebrewInternationalizedContent,
+				weight: "heavy"
+			}, context)
+		})}
+		${Container({
+			direction: "column",
+			heading: "Hebrew (Light)",
+			content: InternationalizedRegularBoldItalicGroup({
+				...args,
+				semantics: "heading",
+				lang: "he",
+				internationalizedContent: HebrewInternationalizedContent,
+				weight: "light"
+			}, context)
+		})}
+	`
+});
+
+export const DocsInternationalizedBodyVariants = (args, context) => Container({
+	withBorder: false,
+	content: html`
+		${Container({
+			direction: "column",
+			heading: "Japanese",
+			content: InternationalizedRegularBoldItalicGroup({
+				...args,
+				semantics: "body",
+				lang: "ja",
+				internationalizedContent: JapaneseInternationalizedContent,
+			}, context)
+		})}
+		${Container({
+			direction: "column",
+			heading: "Arabic",
+			content: InternationalizedRegularBoldItalicGroup({
+				...args,
+				semantics: "body",
+				lang: "ar",
+				internationalizedContent: ArabicInternationalizedContent,
+			}, context)
+		})}
+		${Container({
+			direction: "column",
+			heading: "Hebrew",
+			content: InternationalizedRegularBoldItalicGroup({
+				...args,
+				semantics: "body",
+				lang: "he",
+				internationalizedContent: HebrewInternationalizedContent,
+			}, context)
+		})}
+	`
+});
+
+export const DocsInternationalizedDetailVariants = (args, context) => Container({
+	withBorder: false,
+	content: html`
+		${Container({
+			direction: "column",
+			heading: "Japanese",
+			content: InternationalizedRegularBoldItalicGroup({
+				...args,
+				semantics: "detail",
+				lang: "ja",
+				internationalizedContent: JapaneseInternationalizedContent,
+			}, context)
+		})}
+		${Container({
+			direction: "column",
+			heading: "Arabic",
+			content: InternationalizedRegularBoldItalicGroup({
+				...args,
+				semantics: "detail",
+				lang: "ar",
+				internationalizedContent: ArabicInternationalizedContent,
+			}, context)
+		})}
+		${Container({
+			direction: "column",
+			heading: "Hebrew",
+			content: InternationalizedRegularBoldItalicGroup({
+				...args,
+				semantics: "detail",
+				lang: "he",
+				internationalizedContent: HebrewInternationalizedContent,
+			}, context)
+		})}
+	`
+});
+
+export const DocsInternationalizedCodeVariants = (args, context) => Container({
+	withBorder: false,
+	wrapperStyles: {"row-gap": "0"},
+	content: html`
+	${Container({
+		direction: "column",
+		heading: "Japanese",
+		wrapperStyles: {"row-gap": "0"},
+		content: html`
+				${Template({
+					...args,
+					context,
+					content: ["暗号", "Code"],
```

Comment:
The lang attribute is set in `Template` to be added to the outer `.spectrum-Typography` `<div>` wrapper, but this wrapper only works if there's more than 1 item--this is a little bit hacky but this is the only place where it needs to be used.